tests/Functional/DiffTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Manala package.
+ *
+ * (c) Manala <contact@manala.io>
+ *
+ * For the full copyright and license information, please refer to the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Manala\Tests\Functional;
+
+use Manala\Command\Diff;
+use Manala\Env\EnvEnum;
+use Symfony\Component\Console\Tester\CommandTester;
+
+class DiffTest extends \PHPUnit_Framework_TestCase
+{
+    const ACME_PROJECT_WD = MANALA_DIR.'/tests/fixtures/Command/DiffTest/AcmeProject';
+    const EXPECTED_OUTPUT_FILE = MANALA_DIR.'/tests/fixtures/Command/DiffTest/expected_acme.patch';
```

Comment:
A less expressive but simpler way to test this could be to create a temporary `symfony/framework-standard-edition` project (see [BuildTest](https://github.com/manala/manala/blob/master/tests/Functional/BuildTest.php#L38)), manalize it and edit some ansible files, giving an always predictible diff